content/2023-08-29_pyladies-byly-jsme-na-europythonu.md

```diff
@@ -0,0 +1,57 @@
+Title: PyLadies & Pyvo - Byly jsme na EuroPythonu
+Date: 2023-08-29 18:00:00
+Author: Karolina Surma, Mia Bajic
+
+Do Prahy letos poprvé doputovala konference EuroPython.
+Mezi 17. 7. a 23. 7. 2023 jste tam mohli potkat velkou část české Python komunity.
+EuroPython nám poskytl jak možnost posílit vazby v naší komunitě, tak rozšířit sítě do mezinárodních vod.
```

Comment:
Tahle věta je divná. Je tam vazba *jak možnost, tak rozšířit*. Přidal bych tam jedno slovo navíc:

```suggestion
EuroPython nám poskytl jak možnost posílit vazby v naší komunitě, tak šanci rozšířit sítě do mezinárodních vod.
```